prometheus/promhttp/http.go

```diff
@@ -379,6 +379,9 @@ type HandlerOpts struct {
 	// NOTE: This feature is experimental and not covered by OpenMetrics or Prometheus
 	// exposition format.
 	ProcessStartTime time.Time
+	// If true, Counters, Summaries and Histograms will be exposed with additional 
```

Comment:
If we focus on proto, I think we can skip that and always send CT, no? (: (discussed in the mtg sync)